R/pkg/R/DataFrame.R

```diff
@@ -3184,6 +3200,7 @@ setMethod("histogram",
 #' @param x A SparkDataFrame
 #' @param url JDBC database url of the form `jdbc:subprotocol:subname`
 #' @param tableName The name of the table in the external database
+#' @param ... additional JDBC database connection propertie(s).
```

Comment:
the singular form is property and plural is properties, so the `()` doesn't really work in this case. let's just leave it as properties.